admin/run

```diff
@@ -50,27 +50,28 @@ command_full_path=$(which "$1")
 command_verbatim=$1
 shift
 
+pre_command='carton exec --'
+username="$(id -u -n)"
+if id -u 'musicbrainz' &>/dev/null && [[ "$username" != 'musicbrainz' ]]
+then
+  pre_command="sudo --preserve-env --set-home -u musicbrainz -- $pre_command"
+  username='musicbrainz'
+fi
```

Comment:
There is no installation requirement for having a `musicbrainz` user at the moment. For example `musicbrainz-docker` doesn’t have any even though it really should.

Reply:
This also only runs if the `id -u 'musicbrainz'` check passes, so I don't believe I added any new requirements to the script. If there's no musicbrainz user, it uses whatever name it got from `username="$(id -u -n)"`.